src/devices_models/devices/hydro_generation.jl

```diff
@@ -348,10 +360,14 @@ function add_constraints!(
     X::Type{<:PM.AbstractPowerModel},
 ) where {V <: PSY.HydroPumpedStorage, W <: AbstractHydroReservoirFormulation}
     if get_attribute(model, "reservation")
+        # TODO: Are the "reservation" models compatible with SC FF?
```

Comment:
@sourabhdalvi please check this combination to make sure it actually works.

Reply:
This is for pumped hydro, and we don't have a commitment formulation for pumped hydro. So this reservation formulation is similar to Storage BookKeeping